### PR TITLE
Update dependencies w/ breaking changes

### PR DESCRIPTION
**Pull request by hermann-p**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,58 +46,58 @@
...
-    "react-dom": "15.6.1",
-    "react-dropzone": "4.2.0",
-    "react-hot-loader": "3.0.0",
-    "react-i18next": "2.0.1",
-    "react-loader": "2.4.2",
+    "react-dom": "16.0.0",
+    "react-dropzone": "4.2.1",
+    "react-hot-loader": "3.1.1",
+    "react-i18next": "6.1.0",
+    "react-loader": "https://github.com/Ross146/react-loader/tarball/master",
```
Comment (alexvetter) [inline package.json]: It is necessary because of react >= 16?

File: package.json [hunk 2]
```diff
@@ -46,58 +46,58 @@
...
+    "i18next": "10.0.7",
     "i18next-resource-store-loader": "0.1.2",
-    "js-cookie": "2.1.4",
+    "js-cookie": "2.2.0",
     "keymirror": "0.1.1",
     "lodash": "4.17.4",
     "markdown": "0.5.0",
     "moment": "2.19.1",
     "node-sass": "4.5.3",
+    "patch-package": "^3.6.0",
```
Comment (alexvetter) [inline package.json]: Please remove the `^`.

File: src/main/js/components/header/Spinner.jsx [hunk 3]
```diff
@@ -79,16 +79,18 @@ export default class Spinner extends React.Component {
         </div>
       );
     } else {
-      return null;
+      return <div/>;
```
Comment (alexvetter) [inline src/main/js/components/header/Spinner.jsx]: Why?

File: src/main/js/app.js [hunk 4]
```diff
@@ -64,7 +64,6 @@ if (isProduction) {
       }
     });
 } else {
-  window.Perf = require("react-addons-perf");
   require("../../tests/runTests");
```
Comment (alexvetter) [inline src/main/js/app.js]: Perhaps we should think about running these tests with `mocha` or `jest`. Could be done in a future branch/task. We could than leverage from automated test execution on build time.

File: package.json (hunk 2 above)
Comment (alexvetter) [reply package.json]: Read the README of this package. Looks like it is not used?

File: src/main/scss/cells/cell.scss [hunk 5]
```diff
@@ -100,6 +100,7 @@
     position: relative;
     white-space: nowrap;
     text-overflow: ellipsis;
+    width: 100%;
```
Comment (alexvetter) [inline src/main/scss/cells/cell.scss]: Tested in IE?

File: package.json (hunk 2 above)
Comment (hermann-p) [reply package.json]: Right. Uninstalled it but was left in the .json

File: package.json (hunk 1 above)
Comment (hermann-p) [reply package.json]: It links to a pull request which modifies react-loaders package.json to allow react >= 16

File: src/main/scss/cells/cell.scss (hunk 5 above)
Comment (hermann-p) [reply src/main/scss/cells/cell.scss]: Did not yet have a chance to test on IE.

File: src/main/js/components/header/Spinner.jsx (hunk 3 above)
Comment (hermann-p) [reply src/main/js/components/header/Spinner.jsx]: Else we will have an empty children prop in the portal-hoc, crashing the whole app

File: src/main/js/components/header/ColumnFilter.jsx [hunk 6]
```diff
@@ -48,7 +48,7 @@ class ColumnFilter extends React.Component {
         <a href="#" className={buttonClass} onClick={this.togglePopup}>
           <i className="fa fa-eye" />
           {(nHidden > 0)
-            ? <text className="infotext">{message}</text>
+            ? <span className="infotext">{message}</span>
```
Comment (alexvetter) [inline src/main/js/components/header/ColumnFilter.jsx]: Search & replace `<text>`?

**Comment by alexvetter** [comment]
LGTM, except known bugs (see AC)

**Pull request merged**